src/stores/lock-store-base.js

```diff
@@ -3,20 +3,23 @@ import StoreBase from './store-base.js';
 import uniqueId from '../utils/unique-id.js';
 
 class LockStoreBase extends StoreBase {
-  constructor(collection, expirationTimeInMinutes = null) {
-    super(collection);
-    this.expirationTimeInMinutes = expirationTimeInMinutes;
-  }
+  async takeLock({ type, key, expirationTimeInMinutes }) {
+    const expires = expirationTimeInMinutes ? moment().add(expirationTimeInMinutes, 'minutes').toDate() : null;
+
+    const lock = {
+      _id: uniqueId.create(),
+      type,
+      key,
+      sessionKey: uniqueId.create(),
```

Comment:
I would like to delete this `sessionKey`, as I can't find any usage of it. Do you remember how this was used?

Reply:
I think it was once planned as a means to correlate eventual lingering locks to a certain action, but I think we can delete it, the system has proven reliable so it will never be needed, I guess.

Reply:
Ok I see, thanks for the insight 👍 Removed!